src/main/java/com/google/sps/servlets/AccountData.java

```diff
@@ -41,27 +50,41 @@ public String getLogInOutURL() {
     public String getSignUpURL() {
```

Comment:
Add javadoc noting when this is null.

Same below.

Reply:
added